README.md

```diff
@@ -81,20 +117,40 @@ When using configuration file, run:
 ssv-dkg start-operator --configPath "/examples/config/operator4.example.yaml"
 ```
 
+If the `--configPath` parameter is not provided, `ssv-dkg` will be looking for a file named `operator.yaml` in `./config/` folder at the same root as the binary (i.e. `./config/operator.yaml`)
+
+#### Launch with Docker and YAML file
+
+The team builds a Docker image with every release of the tool.
+
+It is advised to store all the necessary files (encrypted_private_key.json, password) in a single folder (represented below with <PATH_TO_FOLDER_WITH_CONFIG_FILES>) and share such folder with the container as a volume.
+
+To use it, run the command below:
+
+```sh
+docker run --name ssv_dkg -p 3030:3030  \
+-v "<PATH_TO_FOLDER_WITH_CONFIG_FILES>":/data -it \
+"ssv-dkg:latest" start-operator --configPath /data/operator.yaml 
+```
+
 ### Initiator
 
-The initiator uses `init` to create the initial details needed to run DKG between all operators.
+#### Generate Initiator identity RSA key pair
 
-Generate initiator identity RSA key pair:
+The Initiator needs to sign all messages exchanged with DKG participants with an RSA key.
+
+To generate Initiator RSA keys, launch the following command, replacing `<PASSWORD>` with a password of your choosing:
 
 ```sh
-ssv-dkg generate-initiator-keys --password 12345678
+ssv-dkg generate-initiator-keys --password <PASSWORD>
 ```
 
-This will create `encrypted_private_key.json` with encrypted by password RSA key pair
-Write down `password` in any text file, for example to `./password`
+This will create `encrypted_private_key.json` with encrypted by password RSA key pair.
+Write down your chosen password in any text file, for example to `./password`.
```

Comment:
Eventually we will probably point to use the key generation utility in the ssv node, since we don't want to duplicate it.

Reply:
Thanks for the heads-up. When that happens this should be updated, I'll make sure of that.